packages/radix/src/components/GhostButton.tsx

```diff
@@ -55,6 +54,26 @@ export const GhostButton = React.forwardRef<HTMLButtonElement, GhostButtonProps>
           },
         },
         variants: {
+          size: {
+            0: {
+              button: {
+                normal: {
+                  height: theme.sizes[5],
+                  minWidth: theme.sizes[5],
+                },
+              },
+            },
+            1: {
+              button: {
+                normal: {
+                  height: theme.sizes[6],
+                  minWidth: theme.sizes[6],
+                  paddingTop: theme.space[1],
+                  paddingBottom: theme.space[1],
```

Comment:
Is the font size not being increased on purpose?

Reply:
Adding 👌